src/handlers/getQuotes.handler.js

```diff
@@ -0,0 +1,26 @@
+const {fetchFiftyQuote,specificQuote}  = require('../services/getQuotes.service')
+
+const fetchQuotesHandler = async(req,res) =>{
```

Comment:
Why is the file name `getQuotes.handler`? If I want to create quotes, will I have to create another handler?